front-end/src/components/Admin/index.js

```diff
@@ -16,6 +17,12 @@ class Admin extends Component {
     activeKey: '1'
   };
 
+  setRecordData = (data) => {
```

Comment:
I've tried to create a few tabs and open Clients, and client editor and looked in React tab in Chrome dev tools. The recordData property doesn't change.